template.yaml

```diff
@@ -1,40 +1,47 @@
 AWSTemplateFormatVersion: '2010-09-09'
 Transform: AWS::Serverless-2016-10-31
 Description: >
-  Hello World lambda
-
-Metadata:
-  AWS::ServerlessRepo::Application:
-    Name: "lambda-template"
-    Description: "A GH template for quickly starting a new AWS lambda project."
-    Author: "Sage-Bionetworks"
-    SpdxLicenseId: "Apache-2.0"
-    # paths are relative to .aws-sam/build directory
-    LicenseUrl: "LICENSE"
-    ReadmeUrl: "README.md"
-    Labels: ["serverless", "template", "github", "quick-start"]
-    HomePageUrl: "https://github.com/Sage-Bionetworks-IT/lambda-template"
-    SemanticVersion: "0.0.3"
-    SourceCodeUrl: "https://github.com/Sage-Bionetworks-IT/lambda-template/tree/0.0.3"
+  MIPS transform and cache API
 
 # More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
 Globals:
   Function:
-    Timeout: 30
+    Timeout: 10
+
+Parameters:
+  MipsOrganization:
+    Type: String
+    Description: Log in to this organization in the finance system
+  SsmParamPrefix:
+    Type: String
+    Description: Path prefix for secure parameters in SSM
+    Default: '/lambda/mipsSecret'
+  SsmKeyAdminArns:
+    Type: CommaDelimitedList
+    Description: List of role ARNs to manage the encryption key
+  SsmAliasPrefix:
+    Type: String
+    Default: lambda-mips-api
+    Description: Prefix used in the KMS Key Alias name
 
 Resources:
-  HelloWorldFunction:
+  Function:
     Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
     Properties:
       CodeUri: .
-      Handler: hello_world/app.lambda_handler
+      Handler: mips_api.lambda_handler
       Runtime: python3.9
+      Environment:
+        Variables:
+          MipsOrg: !Ref MipsOrganization
+          SsmPath: !Ref SsmParamPrefix
+          apiAllCostCenters: '/all/costcenters.json'
```

Comment:
your thoughts on changing this endpoint to `/costcenters/all` to return all costcenters.  I'm thinking in case we want to return more info about a specific cost center in the future we can create a `/costcenter/$COST_CENTER` endpoint which would return details of the specific $COST_CENTER.

Reply:
I don't see a point to that, there isn't any "more info" from program codes, the only info we get from the finance system is a mapping between program codes and program names.

Reply:
ahh, ok.